api/app.go

```diff
@@ -35,15 +34,6 @@ func NewApp(
 }
 
 func (a *App) configureApp() {
```

Comment:
You should still set the default values somewhere around here, you can't be sure that the received configuration will have values set for the values you use.